### PR TITLE
FIX: Printing to CLI

### PR DESCRIPTION
**Pull request by ecoble**
This includes a fix for printing, previously trying to call print_to_cli more than once between input prompts would cause only the last output to show, because it was overwriting previous input. This PR fixes this by moving the cursor down after each output, instead of at the beginning of each input. This also includes a small change of printing action management's success strings in action operation functions, instead of a generic success message that was being used as a placeholder.

### CODE REVIEW

File: src/ui/src/print_functions.c [hunk 1]
```diff
@@ -122,8 +121,8 @@ void print_to_cli(chiventure_ctx_t *ctx, char *str)
     char temp[len];
     strcpy(temp, str);
     char *tmp = strtok(temp, "\n");
-    
-    
+
+
```
Comment (mpcsh) [inline src/ui/src/print_functions.c]: Can we delete this extra newline

**Comment by mpcsh** [comment]
LGTM but @ecoble please resolve the merge conflict!

**Comment by ecoble** [comment]
CLOSING STATEMENT: This PR fixes an issue were multiple calls to print_to_cli would overwrite the previous one, so only the last call showed up. This has been fixed so all lines of text appear.

Note: This introduced a bug with displaying the help text, which is being fixed on PR #375

**Comment by lydiafilipe** [comment]
Issue Score: ✓+

Comments:
Nice work!

**Pull request merged**